packages/swap/src/event-handlers/v120/networkDepositReceived.ts

```diff
@@ -4,16 +4,20 @@ import logger from '../../utils/logger';
 import { EventHandlerArgs } from '../index';
 
 export const actionSchema = z.union([
-  z.object({ __kind: z.literal('Swap'), swapId: u64 }),
+  z.object({ __kind: z.literal('Swap'), swapId: u128 }),
   z.object({ __kind: z.literal('LiquidityProvision'), lpAccount: hexString }),
   z.object({
     __kind: z.literal('CcmTransfer'),
-    principalSwapId: u64.nullable().optional(),
-    gasSwapId: u64.nullable().optional(),
+    principalSwapId: u128.nullable().optional(),
+    gasSwapId: u128.nullable().optional(),
   }),
   z.object({
     __kind: z.literal('NoAction'),
   }),
+  z.object({
+    __kind: z.literal('BoostersCredited'),
```

Comment:
should the handler do something with this action? right now it seems like handler will just ignore it

Reply:
Thats coming in a follow up PR.

Reply:
The commit message might've been confusing. Will change that.